app/styles/_component-variables.scss

```diff
@@ -901,7 +901,7 @@ body {
   --file-details-api-scan-captured-apis-border-radius: var(--border-radius);
   --file-details-api-scan-captured-apis-card-box-shadow: var(--box-shadow-3);
   --file-details-api-scan-captured-apis-card-background: var(--background-main);
-  --file-details-api-scan-captured-apis-title-background: var(
+  --file-details-api-scan-select-all-captured-api-background: var(
```

Comment:
name should be `--file-details-api-scan-captured-apis-select-all-background`

Reply:
Done